City-of-Helsinki/helsinki-paatokset
# PP-0: Drupal core and module updates for 21.04.2022

**To test:**
- Checkout branch, run `make fresh`
- Check that there are no pending updates: https://helsinki-paatokset.docker.so/fi/admin/reports/updates
- Test basic site functionality:
  - Creating and editing page and landing page content
  - Check decisions and policymaker content, especially sidebar navigation on policymaker pages
- Updates from helfi_base_config:
  - New contact card paragraph should be available on pages and landing pages
  - Filelog module should be enabled
- Check the list of updated modules and see if there's any functionality that needs to be tested

**Updates:**

```
Lock file operations: 4 installs, 28 updates, 0 removals
  - Upgrading behat/mink (v1.9.0 => v1.10.0)
  - Upgrading behat/mink-selenium2-driver (v1.5.0 => v1.6.0)
  - Upgrading commerceguys/addressing (v1.2.2 => v1.3.0)
  - Upgrading composer/composer (2.2.9 => 2.2.12)
  - Upgrading consolidation/annotated-command (4.5.2 => 4.5.4)
  - Upgrading drupal/config_filter (2.2.0 => 2.3.0)
  - Upgrading drupal/core (9.3.9 => 9.3.12)
  - Upgrading drupal/core-composer-scaffold (9.3.9 => 9.3.12)
  - Upgrading drupal/core-dev (9.3.9 => 9.3.12)
  - Upgrading drupal/core-recommended (9.3.9 => 9.3.12)
  - Upgrading drupal/hdbt (2.3.5 => 2.3.14)
  - Upgrading drupal/helfi_api_base (2.1.9 => 2.2.1)
  - Upgrading drupal/helfi_platform_config (2.6.5 => 2.6.19)
  - Upgrading drupal/helfi_tpr (2.0.5 => 2.0.11)
  - Upgrading drupal/matomo (1.17.0 => 1.18.0)
  - Upgrading drupal/paragraphs (1.13.0 => 1.14.0)
  - Upgrading drupal/token_filter (1.3.0 => 1.4.0)
  - Upgrading instaclick/php-webdriver (1.4.10 => 1.4.14)
  - Upgrading josdejong/jsoneditor (v5.29.1 v5.29.1 => v5.29.1 )
  - Upgrading justinrainbow/json-schema (5.2.11 => 5.2.12)
  - Upgrading phpdocumentor/type-resolver (1.6.0 => 1.6.1)
  - Upgrading phpstan/phpdoc-parser (1.3.0 => 1.4.4)
  - Upgrading ruflin/elastica (dev-master 3a19d82 => dev-master 72a4598)
  - Upgrading sebastian/environment (5.1.3 => 5.1.4)
  - Upgrading seld/jsonlint (1.8.3 => 1.9.0)
  - Upgrading slevomat/coding-standard (7.0.20 => 7.1)
  - Upgrading symfony/lock (v4.4.38 => v4.4.40)
  - Upgrading symfony/phpunit-bridge (v5.4.3 => v5.4.7)
  - Locking drupal/views_bulk_edit (2.6.0)
  - Locking drupal/views_bulk_operations (4.1.2)
  - Locking drupal/media_entity_file_replace (1.0.0)
  - Locking drupal/filelog (2.1.0)
```
